azureadps-1.0/MSOnline/Get-MsolDevice.md

```diff
@@ -88,6 +97,20 @@ PS C:\>Get-MsolDevice -RegisteredOwnerUpn "pattifuller@contoso.com"
 
 This command gets all the devices registered by the user with the UPN named pattifuller@contoso.com.
 
+### Example 7: Get device by activity (logon) timestamp
+```
+PS C:\>Get-MsolDevice -LogonTimeBefore 'January 1, 2017 12:00:00 AM'
```

Comment:
You need to include the -All flag here:
`Get-MsolDevice -All -LogonTimeBefore 'January 1, 2017 12:00:00 AM'`